src/main/resources/application.properties

```diff
@@ -89,3 +89,6 @@ demand.request.notification.template=demandCreatetRequestTemplate
 
 announcement.default.search.pageSize=15
 
+sb.search.service.host=http://192.168.1.5:6007
+sb.composite.v4.search=/v4/search
+playlist_course_facets=courseCategory
```

Comment:
No need to get this from property... 
courseCategory is attribute -- this attribute exists in all course... value would change... since we are not searching for values, no need to put this in config.

Reply:
We are using this in future if they want more factes info no need to change at code level and from this config we get this value @karthik-tarento